pkg/kyma/cmd/install/cmd.go

```diff
@@ -77,14 +77,9 @@ func NewCmd(o *Options) *cobra.Command {
 	cobraCmd := &cobra.Command{
 		Use:   "install",
 		Short: "Installs Kyma on a running Kubernetes cluster",
-		Long: `Install Kyma on a running Kubernetes cluster.
-
-Make sure that your KUBECONFIG is already pointing to the target cluster.
-The command:
-- Installs Tiller
-- Deploys the Kyma Installer
-- Configures the Kyma Installer using the latest minimal configuration
-- Triggers Kyma installation
+		Long: `Installs Kyma on a running Kubernetes cluster.
+
```

Comment:
PLease add the link already. As discussed there is no governance job for source code

Reply:
Added :)